src/currency/currency_service.bal

```diff
@@ -0,0 +1,56 @@
+import ballerina/grpc;
+import ballerina/io;
+import ballerina/lang.'decimal as decimal;
```

Comment:
```suggestion
import ballerina/lang.'decimal as decimal;
```

I wonder, how this is possible since `decimal` is a keyword. Better to rename it to improve the readability.